app/controllers/invite_controller.rb

```diff
@@ -187,4 +187,21 @@ def testing_is_live?
       end
       return false
     end
+
+    def find_testers_group app
+      group_name = ENV["ITC_EXTERNAL_GROUP"]
+      if group_name.present?
+        group = Spaceship::TestFlight::Group.find(app_id: app.apple_id,
```

Comment:
BTW `app` definition should be actually extracted into a method but I thought it's out of the scope of this PR.